README.md

```diff
@@ -1,51 +1,72 @@
 # 3D Toolkit
 
-A toolkit for building powerful stereoscopic 3d experiences that run on azure
-and stream to devices. :muscle: :eye: :cloud:
+A toolkit for building powerful stereoscopic 3d experiences that run on the cloud and stream to devices. :muscle: :eye: :cloud:
 
-> Note: If you are using Visual Studio 2017, ensure you have installed the v140 c++ build tools, and please __do not update our projects when prompted to do so__.
+## Features
+
+> See our client and server implementation feature matrices [here](https://github.com/CatalystCode/3dtoolkit/wiki/Feature-matrices)!
+
++ WebRTC powered audio/video content streaming
++ Client user input streaming
++ Low-latency communication channels
++ Secure communication channels
++ Client user authentication
++ Server application authentication
++ Sample implementations for servers and clients
 
 ## Grabbing latest prebuilt binaries
 
+> These binaries are prebuilt toolkit binaries - if you're looking to contribute, or build your own, keep reading! 
+
 [CI Builds](https://3dtoolkitstorage.blob.core.windows.net/builds/index.html) - Not recommended
 
 [Pull Request CI Builds](https://3dtoolkitstorage.blob.core.windows.net/pullrequests/index.html) - Not recommended
 
 [Release/Stable Tagged Builds](https://3dtoolkitstorage.blob.core.windows.net/releases/index.html)
 
-## Prerequisites
+## Prerequisite Tooling
 
 + Windows 10 Anniversary Update / Windows Server 2012 R2 / Windows Server 2016
 + Visual Studio 2015 Update 3
 + Windows 10 SDK - 10.0.14393.795
-+ [Windows 10 DDK](https://msdn.microsoft.com/en-us/library/windows/hardware/ff557573(v=vs.85).aspx) - If building WebRTC Library from source
++ [Windows 10 DDK](https://msdn.microsoft.com/en-us/library/windows/hardware/ff557573(v=vs.85).aspx) - If building WebRTC Library from source (note: this is likely not the case, and is only necessary if you're planning to modify our dependency library, WebRTC)
```

Comment:
Is it Windows 10 DDK or Windows Driver Kit (WDK) 10?

Reply:
i've never built these myself - maybe @anderm can confirm? my guess is this should be the `WDK` instead, since that's what that links to.